doc/projects.rst

```diff
@@ -2,29 +2,28 @@
 Projects
 ########
 
-Spyder allows users to associate a given directory with a **Project**, which offers several main advantages:
+Spyder allows you to associate a given directory with a **Project**, which automatically saves and restores your :doc:`editor` panes and open files to exactly how you left them in your last session.
 
-* Opening, closing or switching to a :guilabel:`Project` automatically saves and restores your :doc:`editor` panes and open files to exactly how you left off.
-  This allows you to easily switch between many different development tasks without having to manually re-create your session for each one.
-* Your project's root directory is automatically added to the ``PYTHONPATH``, so you can easily ``import`` and work with any modules and packages you create with zero setup.
-* The project path is also used to automatically set your working directory, and can be used as an automatic preset for several modules, such as the :doc:`findinfiles` search location.
-* You can browse all your :guilabel:`Project` files from the :ref:`Project Explorer <project-explorer>`, regardless of your current working directory or :doc:`fileexplorer` location.
-* :guilabel:`Projects` are :ref:`integrated <vcs-section>` with the ``git`` version control system, allowing you to commit files and open them or your repository in the ``gitk`` GUI right from within Spyder.
+With the :ref:`Project <project-explorer>` pane, you can browse all your project's files, regardless of your current working directory or :doc:`fileexplorer` location.
+
+(Screenshot of Spyder with project explorer and projects menu)
+
+In addition, your project's root folder is used to set your working directory, and automatically added to the ``PYTHONPATH``, so you can easily ``import`` and work with any modules and packages you create.
```

Comment:
```suggestion
In addition, your project's root folder is used to set your working directory, and automatically added to the ``PYTHONPATH``, so you can easily ``import`` and work with any modules and packages you create as part of it.
```

Small clarification.

Reply:
```suggestion
In addition, your project's root folder is used to set your working directory, and automatically added to the ``PYTHONPATH``, so you can easily ``import`` and work with any modules and packages you create inside of it.
```

Its not explicit what "as part of it" means with respect to their project, and in any case the "it" here refers to the working directory, so this clarifies that.